web/frontend/src/components/camera/index.svelte

```diff
@@ -33,6 +33,7 @@ const handleRefreshInput = (name: string) => {
     }
 
     refreshFrequency[name] = event.detail.value;
+    triggerRefresh = !triggerRefresh; // todo: there are multiple ways to get here ...
```

Comment:
Why do we want to trigger refresh here?

Reply:
I don't know how eventing is supposed to work, so take all of this with a grain of salt:

it looks like editing `refreshFrequency` doesn't call `updateCameraRefresh` -- update refresh is only called in camera.svelte in the useConnect hook and in the `lastTriggerRefresh` check

that means that before, switching between video + 1s refresh was awkward:
- video -> 1s: first time you switch, you have to hit 'refresh' to start the 1s interval. second time, it works because 
- 1s -> video: the 1s refresh was still happening behind the scenes (you can see it swapping out the url on the <img> element under the video)

what does work: the `{#if refreshRate === 'Live'}` around the LiveCamera does get rid of the element, and seems to be properly reactive. I believe that's happening through `refreshRate={refreshFrequency[camera.name]}` on the Camera element in camera/index.svelte

camera/index.svelte is doing something with triggerRefresh (it has a `{triggerRefresh}` in the markup section, but I'm not sure what it's doing). but it's possible triggerRefresh is getting redeclared and not shared properly? again, not totally sure how to read this yet

caveats: I'm on firefox, may work differently on other browsers. not sure if there's a better way for the `refreshFrequency` map to alert updateCameraRefresh. also not sure how streams are controlled on the RDK side -- like is this properly shutting down the stream, or is RDK still doing encoding work? (this may happen inside LiveCamera; didn't look)

Reply:
now that I read more, a simpler fix could be to change:
```diff
- if (lastTriggerRefresh !== triggerRefresh)
+ if (lastTriggerRefresh !== triggerRefresh || lastRefreshRate !== refreshRate)
```

Reply:
This all SGTM - we'll smoke test this in the AppImage. Thanks!